Leaflet.CheapLayerAt.js

```diff
@@ -22,6 +26,42 @@ L.Map.include({
 		return this._getLayerFromDOMElement(el);
 	},
 
+	getLayersAt: function(point, y) {
+		point = L.point(point, y);
+
+		// Ignore points outside the map
+		if (!this.getSize().contains(point)) { return; }
+
+		var mapPos = this._container.getBoundingClientRect();
+
+		var viewportPoint = L.point(mapPos.left, mapPos.top).add(point);
+
+		var els = this._getElementsFromPoint(viewportPoint.x, viewportPoint.y);
+		var out = [];
+		for(var i = 0; i < els.length; i += 1) {
+			var lay = this._getLayerFromDOMElement(els[i]);
+			if(lay) out.push(lay);
+		}
+		return out;
+	},
+
+	_getElementsFromPoint: function(x, y) {
+		var stack = [], el;
+		var i = 1000; // prevent from infinite loop
```

Comment:
Seems arbitrary. What are the cases where an infinite loop triggers?

Reply:
Browser which not support `pointer-events:none` (eg: IE6-10).
https://caniuse.com/#search=pointer-events

This function was reference the answer on stackoverflow:
https://stackoverflow.com/questions/21051084/javascript-know-all-the-elements-under-your-mouse-pointer-multiple-z-axis-laye
Should I put this link in code comment?

Reply:
Then perform feature detection with something like `if ('pointerEvents' in anyElement.style)` or whatever, or check if the element detected is already at the top of the stack.